### DISCUSSION TITLE
Hey, where is our Preview 1?

### DISCUSSION THREAD

**Discussion by Happypig375** [dotnet/maui#352] [Q&A] [answered]
https://github.com/dotnet/maui/wiki/Roadmap
> ## .NET MAUI Preview 1 (Late 2020)

If even the first previews are not here, can we actually trust the team to deliver .NET MAUI with .NET 6?

**Comment by davidortinau** [comment, marked as answer]
Preview 1 of .NET 6 includes Android and iOS SDKs, the foundation of .NET MAUI. Code for UI is here and PRs in review on the Xamarin.Forms repo. 

Preview 2 will have basic usable pieces for UI to get a clearer look at the architecture of handlers in practice. I encourage you to look to Preview 4 for a more complete release of .NET MAUI. 

Check the project boards and roadmap for updates.

**Comment by Coding201212** [comment]
@davidortinau Will the Preview 2 be published after 6 weeks?
